content/en/serverless/installation/nodejs.md

```diff
@@ -245,66 +244,80 @@ See the [latest release][3].
 
-### Monitor Custom Business Metrics
+### Monitor Custom Business Logic
 
-If you would like to submit a custom metric or manually instrument a function, see the sample code below:
+If you would like to submit a custom metric or span, see the sample code below:
 
 ```javascript
-const { sendDistributionMetric } = require("datadog-lambda-js");
+const { sendDistributionMetric, sendDistributionMetricWithDate } = require("datadog-lambda-js");
 const tracer = require("dd-trace");
 
-// Submit a custom APM span named "sleep"
+// submit a custom span named "sleep"
 const sleep = tracer.wrap("sleep", (ms) => {
   return new Promise((resolve) => setTimeout(resolve, ms));
 });
 
 exports.handler = async (event) => {
-  await sleep(1000);
+  // add custom tags to the lambda function span,
+  // does NOT work when X-Ray tracing is enabled
+  const span = tracer.scope().active();
+  span.setTag('customer_id', '123456');
 
-  // Submit a custom metric
+  await sleep(100);
+
+  // submit a custom span
+  const sandwich = tracer.trace('hello.world', () => {
+    console.log('Hello, World!');
+  });
+
+  // submit a custom metric
   sendDistributionMetric(
     "coffee_house.order_value", // metric name
-    12.45, // metric Value
+    12.45, // metric value
+    "product:latte", // tag
+    "order:online", // another tag
+  );
+
+  // submit a custom metric with timestamp
+  sendDistributionMetricWithDate(
+    "coffee_house.order_value", // metric name
+    12.45, // metric value
+    new Date(Date.now()), // date, must be within last 20 mins
     "product:latte", // tag
     "order:online", // another tag
   );
+
   const response = {
     statusCode: 200,
     body: JSON.stringify("Hello from serverless!"),
   };
   return response;
 };
 ```
-[Enable custom metric submission][3] to get started.
-
-### Enable the AWS X-Ray Integration
-
-Datadog’s integration with AWS X-Ray allows you to visualize end-to-end serverless transactions, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, [choose between setting up Datadog APM or the AWS X-Ray integration][4] for your tracing needs.
 
-{{< img src="integrations/amazon_lambda/lambda_tracing.png" alt="Architecture diagram for tracing AWS Lambda with Datadog" >}}
+For more information on custom metric submission, see [here][4]. For additional details on custom instrumentation, see the Datadog APM documentation for [custom instrumentation][5].
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /integrations/amazon_web_services/
 [2]: /serverless/forwarder
-[3]: /serverless/custom_metrics
-[4]: /serverless/distributed_tracing
+[3]: https://app.datadoghq.com/functions
+[4]: /serverless/custom_metrics?tab=nodejs
+[5]: https://docs.datadoghq.com/tracing/custom_instrumentation/nodejs/
```

Comment:
```suggestion
[5]: /tracing/custom_instrumentation/nodejs/
```

Reply:
Thanks, I think we have similar issue in other files too, will fix them all.